test/config.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  ChromiumEnv,
+  FirefoxEnv,
+  WebKitEnv,
+  test,
+  setConfig,
+  PlaywrightOptions,
+  Config,
+  globalSetup,
+} from "@playwright/test"
+import * as crypto from "crypto"
+import path from "path"
+import { PASSWORD } from "./utils/constants"
+import * as wtfnode from "./utils/wtfnode"
+
+// Playwright doesn't like that ../src/node/util has an enum in it
```

Comment:
:-1:  that you had to do this, but :+1:  that you added a comment... I could certainly be better about doing this myself, it's a great habit :smile: